File: Codewars/java/src/main/java/codewars/FindSquares.java
package codewars;

public class FindSquares {

    public static String findSquares(int n){
        long start = (long) Math.sqrt(n);
        long end = (long) Math.pow(n, 2);
        for (long i = start; i <= end; i++) {
            if (i*i - (i-1)*(i-1) == n) {
                return i*i + "-" + (i-1)*(i-1);
            }
        }
        return "no-answer";
    }
}

File: Codewars/python/Find_Squares.py
import math


def find_squares(num):
    start = int(math.sqrt(num))
    end = int(math.pow(num, 2))
    for i in range(start, end+1):
        if i*i - (i-1)*(i-1) == num:
            return f'{i*i}-{(i-1)*(i-1)}'

    return "no-answer"


print(find_squares(1))
print(find_squares(9))
print(find_squares(5))
print(find_squares(7))
